### ISSUE TITLE
Support for ElasticSearch internal LoadBalancer

### ISSUE THREAD

**Issue by victoryNap** [bitnami/kube-prod-runtime#797]
tl;dr I'd like to run a single instance of BKPR for several clusters, instead of having one deployed for each.

How, using the config, can I specify that I want ElasticSearch to be available as an internal load balancer in GKE? I have edited the definition in the cluster, but I'd like to maintain it throughout releases. I couldn't find any good documentation on modifying the configuration in such a way.

Looks like its only meant to run as a ClusterIP:
https://github.com/bitnami/kube-prod-runtime/blob/master/manifests/components/elasticsearch.jsonnet#L227-L234

AFAIK this doesn't easily support a consolidated BKPR deployment.

**Comment by javsalgar** [comment]
Hi,

Using jsonnet you can create your custom overrides file, as explained here: https://github.com/bitnami/kube-prod-runtime/blob/master/docs/overrides.md

With that, you could change the service configuration so it uses LoadBalancerIP if that's what you require. Would that work for your case?

**Comment by victoryNap** [comment]
@javsalgar 
I guess my issue is less with BKPR, and more with jsonnet.

Since the ClusterIP is declared in the initial manifest, is there a way to remove that key using config overrides? The spec for the elastic service is largely compiled from other sources, so overriding it completely won't work either.

**Comment by victoryNap** [comment]
Solved the issue with the following overrides:
```            
spec+: {
              clusterIP: "",
              type: "LoadBalancer"
            }
```
Seems like a bit of a hack, but it worked.

Thanks for the help!
